pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -189,6 +188,7 @@ export class DialogFooter extends React.Component {
                 className="apply"
                 variant={ action.style || "secondary" }
                 onClick={ this.action_click.bind(this, action.clicked) }
+                isLoading={ this.state.action_in_progress || action.loading }
```

Comment:
Doesn't this mean that the spinner is in _all_ action buttons? Ideally, it should be only in the one that created the current action promise.